posthog/session_recordings/session_recording_api.py

```diff
@@ -1608,16 +1597,18 @@ def current_user_viewed(recording_ids_in_list: list[str], user: User | None, tea
     return viewed_session_recordings
 
 
+@tracer.start_as_current_span("safely_read_modifiers_overrides")
 def safely_read_modifiers_overrides(distinct_id: str, team: Team) -> HogQLQueryModifiers:
     modifiers = HogQLQueryModifiers()
 
     try:
-        groups = {"organization": str(team.organization.id)}
+        groups = {"organization": str(team.organization_id)}
```

Comment:
calling `team.organization.id` causes a query for the entire organization model if it hasn't previously been loaded
(it's fast but unnecessary in the span i saw https://traces.prod-us.posthog.dev/trace/b19303e3ea89c14de13dacc04a64bffb)
since we only need the id and the team model already knows that we can just access it directly